incubator/zookeeper/README.md

```diff
@@ -1,100 +1,178 @@
-| `ImagePullPolicy`       | Container pull policy              | `Always`                                                   |
-| `Replicas`              | k8s petset replicas                | `3`                                                        |
-| `Component`             | k8s selector key                   | `zk`                                                       |
-| `Cpu`                   | container requested cpu            | `100m`                                                     |
-| `Memory`                | container requested memory         | `512Mi`                                                    |
-| `PeerPort`              | k8s service port                   | `2888`                                                     |
-| `LeaderElectionPort`    | Container listening port           | `3888`                                                     |
-| `Storage`               | Persistent volume size             | `1Gi`                                                      |
-
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+## Configuration
+You can specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
 
-Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. 
+For example,
 
 ```bash
 $ helm install --name my-release -f values.yaml incubator/zookeeper
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+### Resources
+The configuration parameters in this section control the resources requested and utilized by the ZooKeeper ensemble.
+
+| Parameter | Description | Default |
+| --------- | ----------- | ------- |
+| `Servers` | The number of ZooKeeper servers. This should always be (1,3,5, or 7) | `3` |
+| `MinAvailable` | The minimum number of servers that must be available during evictions. This should in the interval `[(Servers/2) + 1,(Servers - 1)]`. | `2` |
+| `Cpu` | The amount of CPU to request. As ZooKeeper is not very CPU intensive, `2` is a good choice to start with. | `2` |
+| `Heap` | The amount of JVM heap that the ZooKeeper servers will use. As ZooKeeper stores all of its data in memory, this value should reflect the size of your working set. The JVM -Xms/-Xmx format is used. |`2G` |
+| `Memory` | The amount of memory to request. This value should be at least 2 GiB larger than `Heap` to avoid swapping. You many want to use `1.5 * Heap` for values larger than 2GiB. The Kubernetes format is used. |`2Gi` |
+| `Storage` | The amount of Storage to request. Even though ZooKeeper keeps is working set in memory, it logs all transactions, and periodically snapshots, to storage media. The amount of storage required will vary with your workload, working memory size, and log and snapshot retention policy. | `250Gi`|
```

Comment:
250Gi is a bit aggressive for an out of the box value can we bring this into the 2 digit range.